server/service/apple_mdm.go

```diff
@@ -606,12 +606,20 @@ func getMDMAppleProfilesSummaryEndpoint(ctx context.Context, request interface{}
 	return &res, nil
 }
 
-func (svc *Service) GetMDMAppleProfilesSummary(ctx context.Context, teamID *uint) (*fleet.MDMAppleConfigProfilesSummary, error) {
+func (svc *Service) GetMDMAppleProfilesSummary(ctx context.Context, teamID *uint) (*fleet.MDMProfilesSummary, error) {
 	if err := svc.authz.Authorize(ctx, fleet.MDMConfigProfileAuthz{TeamID: teamID}, fleet.ActionRead); err != nil {
 		return nil, ctxerr.Wrap(ctx, err)
 	}
 
-	ps, err := svc.ds.GetMDMAppleHostsProfilesSummary(ctx, teamID)
+	ac, err := svc.ds.AppConfig(ctx)
+	if err != nil {
+		return nil, ctxerr.Wrap(ctx, err)
+	}
+	if !ac.MDM.EnabledAndConfigured {
+		return &fleet.MDMProfilesSummary{}, nil
+	}
```

Comment:
That's fine as is, but just to let you know in case you missed it, there's `svc.VerifyMDMWindowsConfigured(ctx)` (and Apple) that does this check and returns an error if it is not configured (in your case it seems we don't want to return an error if it is not configured, but you could detect that and still return the empty list).